app/scripts/modules/core/src/managed/ArtifactRow.module.css

```diff
@@ -50,13 +68,23 @@
 
 .stages {
   display: flex;
-  height: 6px;
-  padding-bottom: 2px;
+  position: absolute;
+  height: 8px;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 1;
+  transition: height 150ms ease-in-out;
 }
 
 .stage {
+  display: flex;
   flex: 1 1 auto;
+  font-size: 12px;
+  color: rgba(0, 0, 0, 0.5);
+  border: 1px solid rgba(0, 0, 0, 0.5);
```

Comment:
@gcomstock I was guessing this for the environment bar borders / text color based on eyeballing the mocks — please LMK if this is not representative of what you were actually using

Reply:
Yep 50% opacity black is exactly what I had in the mocks